cargo-quickinstall/src/lib.rs

```diff
@@ -178,6 +178,7 @@ fn curl_head(url: &str) -> Result<Vec<u8>, InstallError> {
         .arg("--silent")
         .arg("--show-error")
         .arg("--fail")
+        .arg("--location")
```

Comment:
This is unrelated, but I had a failing test on my PR and this is the fix (another fix would be to point it at the right repo so there aren't any redirects in the failure case - we should definitely do that too, and make a release. --dry-run is very niche though, so let's not worry about it being broken for now).